### PR TITLE
PADV-1892 - Classes page and Course Details page are now working correctly.

### PR DESCRIPTION
**Pull request by sergivalero20**
### Ticket

https://agile-jira.pearson.com/browse/PADV-1892

### Description

This PR solves two errors which are occurring in Classes and /Courses/Classes.

1. Classes page: When page is reloaded then all classes are shown and it should be filtered by institution.
2. Courses detail page: Course info is disappearing when page is reloaded.

### Changes made

- [x] Classes page
- [x] Courses details page  

### How to test

- Clone the Institution Portal MFE
- Run npm install
- Run npm run start
- Go to /classes and reload page to make sure everything is working as expected.
- Go to /Courses/Classes and reload page to make sure everything is working as expected. Course info should be displayed correctly.

### CODE REVIEW

File: src/features/Classes/ClassesPage/index.jsx [hunk 1]
```diff
@@ -25,28 +25,18 @@ const ClassesPage = () => {
...
-
-  useEffect(() => {
-    if (queryNotInstructors === 'null' && !resetFiltersRef.current) {
-      dispatch(fetchClassesData(selectedInstitution.id, currentPage, '', instructorsNull));
-    } else if (queryNotInstructors === 'null' && resetFiltersRef.current) {
-      dispatch(fetchClassesData(selectedInstitution.id, currentPage, '', stateClasses.filters));
-    } else {
-      dispatch(fetchClassesData(selectedInstitution.id, currentPage, '', stateClasses.filters));
-    }
-  }, [currentPage]); // eslint-disable-line react-hooks/exhaustive-deps
```
Comment (01001110J) [inline src/features/Classes/ClassesPage/index.jsx:49]: Question here, why is this being eliminated?

**Review by 01001110J — COMMENTED** [review]
Honestly I'm not understanding exactly what's going on here, so how can I reproduce the error/s?

Can you provide a better description of the before and after?

File: src/features/Courses/CoursesDetailPage/index.jsx
Comment (AuraAlba) [inline src/features/Courses/CoursesDetailPage/index.jsx:1]: In this page I get two calls to courses services, I don't know why is that necessary, check if you can remove one of them

**Review by AuraAlba — CHANGES_REQUESTED** [review]

File: src/features/Classes/ClassesPage/index.jsx (hunk 1 above)
Comment (sergivalero20) [reply src/features/Classes/ClassesPage/index.jsx:49]: @01001110J better description was mentioned in the PR letter.

It's being deleting due it's calling two times to the Backend.

**Review by 01001110J — APPROVED** [review]

File: src/features/Courses/CoursesDetailPage/index.jsx
Comment (sergivalero20) [reply src/features/Courses/CoursesDetailPage/index.jsx:1]: @AuraAlba change applied!

**Review by AuraAlba — APPROVED** [review]
